agents/trainer/task.py

```diff
@@ -243,23 +240,22 @@ def gcs_upload(local_dir, gcs_out_dir):
     blob.upload_from_filename(local_file_path)
 
 
-def main(unused_argv):
+def main():
   """Run training."""
   tf.logging.set_verbosity(tf.logging.INFO)
 
   if FLAGS.debug:
     tf.logging.set_verbosity(tf.logging.DEBUG)
 
-  run_config = tf.contrib.learn.RunConfig()
+  tf.contrib.learn.RunConfig()
 
   log_dir = FLAGS.logdir
 
-  agents_config = _get_agents_configuration(
-      FLAGS.hparam_set_id, log_dir, run_config.is_chief)
```

Comment:
Good call yeah _get_agents_configuration is no longer getting the hparam object dynamically from globals() so this simplification is good.